src/MoBi.Core/Serialization/Xml/Serializer/MoBiSimulationXmlSerializer.cs

```diff
@@ -16,7 +16,7 @@ public MoBiSimulationXmlSerializer() : base(SerializationConstants.MoBiSimulatio
       public override void PerformMapping()
       {
          base.PerformMapping();
-         Map(x => x.Results);
+         Map(x => x.ResultsDataRepository);
```

Comment:
This is not good here. You are breaking serialization

Reply:
Not sure how you mean. Let's briefly discuss this today.